react/modules/sample/src/components/BoundaryFilter.js

```
@@ -0,0 +1,773 @@
+    let newSelectedOptions = {};
+    if (removedCodes.length > 0) {
+      newBoundaryOptions = cleanLowerLevels(boundaryType, removedCodes, { ...boundaryOptions });
+      newSelectedOptions = cleanLowerLevelsForSelectedValues(boundaryType, removedCodes, [...selectedValues]);
+    }
+    // Reset removedCodes after processing
+    else {
+      newBoundaryOptions = updatedOptions;
+      newSelectedOptions = selectedValues;
+    }
+    debugger;
+    console.log("newSelectedOptions", newSelectedOptions);
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)